.repo/sample-list-generator/src/constants.ts

```diff
@@ -0,0 +1,16 @@
+export type AvailableFolderTypes = (typeof AVAILABLE_FOLDERS)[number]['type'];
```

Comment:
Thoughts on just defining this as `"API_SAMPLE" | "FUNCTIONAL_SAMPLE"`? I think that's better, because otherwise a typo in `AVAILABLE_FOLDERS` will cause a new type to suddenly appear.

I would also suggest renaming this to just `FolderType`.